packages/app/ios/Orbit/IngestEventEmitter.h

```diff
@@ -0,0 +1,30 @@
+//
+//  EventEmitter.h
+//  Orbit
+//
+//  Created by Ozzie Kirkby on 2022-05-07.
+//
+
+#ifndef EventEmitter_h
+#define EventEmitter_h
+
+#import <React/RCTBridgeModule.h>
+#import <React/RCTEventEmitter.h>
+
+typedef enum {
+    kSuccess,
+    kFailure,
+    kWaiting
+} IntentState;
```

Comment:
You asked for Obj-C style notes, so you'll get them! Feel free to ignore these…

Idiomatic ObjC enum naming would be:

```
typedef enum {
  IntentStateSuccess,
  IntentStateFailure,
  IntentStateWaiting
} IntentState
```

This is because Obj-C doesn't have the TS syntax of `EnumName.Value`—it's just `Value`. The prefix makes for nice autocomplete and readability. See e.g. `UIImageResizingMode`.